ui/component/router/view.jsx

```diff
@@ -152,8 +153,9 @@ function AppRouter(props: Props) {
 
       <Route path={`/`} exact component={HomePage} />
       <Route path={`/$/${PAGES.DISCOVER}`} exact component={DiscoverPage} />
-      <Route path={`/$/${PAGES.AUTH}`} exact component={SignInPage} />
-      <Route path={`/$/${PAGES.AUTH}/*`} exact component={SignInPage} />
+      <Route path={`/$/${PAGES.AUTH_SIGNIN}`} exact component={SignInPage} />
+      <Route path={`/$/${PAGES.AUTH}`} exact component={SignUpPage} />
```

Comment:
Not super sure about this but `signup` will be the "default" authentication page. If you have to enter your wallet password, or need rewards approval, you will be redirected to `/$/signup`